build/windows/gh.wixproj

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <Configuration>Release</Configuration>
+        <Platform Condition="'$(Platform)' == ''">x64</Platform>
```

Comment:
Not only. In gh.wxs you need another conditional block like I did [here](https://github.com/PowerShell/openssh-portable/pull/521/files#diff-782d036a41909a866678f429a9771d729a2c6377c661f499aa1707efad24f17bR8-R16), with different UpgradeCodes of course. I can add that now if you think you might release an x86 version in the future, or even prepare it for an ARM64 release, though you'd need to install a newer version of which - and x86 will run fine on ARM in Windows (an EXE rarely needs different bitnesses, but I think the problem people were trying to avoid here is Windows' automatic file system redirection by default).

Reply:
Whether you use it or not, I went ahead and added support for other architectures. Also moved the `<Environment>` authoring to the file component to avoid having to set a component GUID explicitly. It's valid authoring, as the component is the lowest unit of install and tying the file together with the environment that effectively points to it makes sense.